jwst/master_background/master_background_step.py

```diff
@@ -73,7 +78,53 @@ def process(self, input):
                 self.record_step_status(result, 'master_background', success=False)
 
                 return result
+            # Check if background was subtracted in calspec 2 and force_subtract is False
+            # first check if input is a model container
+
+            do_sub = True  # flag if set to False then no master background subtraction is done
+            if not self.force_subtract:  # default mode
+
+                # check if the input data is a model container. If it is then loop over
+                # container and see if the background was subtracted in calspec2.
+                # If all data was  background subtracted. Print log.info and skip master bgk subtrction
+                # If there is a mixture of some being background subtracted print, warning
+                # and message to user to use force_subtract to force the master background
+                # to be subtracted.
+                if isinstance(input_data, datamodels.ModelContainer):
+                    isub = 0
+                    for indata in input_data:
+                        if indata.meta.cal_step.back_sub == 'COMPLETE':
+                            do_sub = False
+                            isub += 1
+
+                    if not do_sub and isub == len(input_data):
+                        self.log.info(
+                            "Not subtracting master background, background was subtracted in calspec2")
+                        self.log.info("To force the master background to be subtracted from this data,"
```

Comment:
This is a really minor comment, but you could add a space after the comma.  The same comment applies to lines 110 and 118 as well.